src/ui/common/src/components/workflows/workflowHeader.tsx

```diff
@@ -286,7 +286,7 @@ const WorkflowHeader: React.FC<Props> = ({ user, workflowDag, workflowId }) => {
           <Status status={workflow.dagResults[0].status} />
         </Box>
 
-        <Box sx={{ ml: 2 }}>
+        <Box sx={{ mr: 20 }}>
```

Comment:
Do we need to worry about this hard-coded "20"?

Reply:
The 2 was using the built in MUI theme spacing. Presumably there aren't 20 spacing increments. Is this defaulting to 20 pixels, @agiron123?

Reply:
@vsreekanti nah, this is 160 pixels.

I wouldn't worry about it for now, this stuff will change during the ui-redesign merge most likely.